service/plugin/include/pluginservice/pluginservice.h

```diff
@@ -22,6 +22,8 @@ class PluginServiceHandler : virtual public PluginServiceIf
 
   void getPlugins(std::vector<std::string>& return_) override;
 
+  void getThriftPlugins(std::vector<std::string> & _return) override;
+
```

Comment:
This separation was required, as the thrift generated JS resources cannot be loaded with the Dojo bundle, while the custom JavaScript files are Dojo-compliant. Therefore it was very inconvenient that the `getWebPlugins()` method returned both collection of files.